### DISCUSSION TITLE
Adding a highlight.js language (mirc)

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#2846] [Help / Questions]
Hello!

Question / help needed with highlight.js if you can!

How do I add [this file](https://raw.githubusercontent.com/highlightjs/highlightjs-mirc/master/mirc.js) and make it work with:
````
```mirc
alias hello {
  echo -ag hello world!
}
```
````
I've tried to copy the file to `/path/to/wikijs/node_modules/highlight.js/lib/languages/mirc.js` as well as amend the index.js file found in `/path/to/wikijs/node_modules/highlight.js/lib/index.js` and add the line:
`hljs.registerLanguage('mirc', require('./languages/mirc'));` but it's not syntax highlighting...

regards

Sven

OS: Debian 10
Wiki.js 2.5.170
Database engine: psql (11.9 (Debian 11.9-0+deb10u1))

**Comment by NGPixel** [comment]
You would need to recompile the client assets.

https://docs.requarks.io/dev#building-production-assets

**Comment by ghost** [comment]
Alright, thanks!

I see that I need to issue a `yarn build` command, but I installed wiki.js following these instructions: https://docs.requarks.io/install/linux
if I install yarn via `npm i -g yarn` and `apt-get install build-essentials` then issue the `yarn build` command, I run into all sort of troubles. I'm very new at this :-) - this might just be over my head...
